djangocms_references/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class References(models.Model):
+    """Dummy model used for permission handling. No DB tables are
+    actually created.
+    """
```

Comment:
You sure this is preferable to a data migration? Cause I don't much like the idea of a model class existing (even with `managed = False`) solely so we can populate some data in a different table.

Reply:
I think it's more clean than creating a Permission that's detached from any model. Besides, a Permission has to be attached to a ContentType, so we'd have to create a dummy ContentType anyway?